<launch>
  <!-- 
       Using simulation time means nodes initialized after this
       will not use the system clock for its ROS clock and 
       instead wait for simulation ticks. 

       See http://wiki.ros.org/Clock

       Note: set to false for deploying to a real robot.
  -->
  <arg name="use_sim_time" default="true"/>
  <param name="use_sim_time" value="$(arg use_sim_time)"/>

  <!-- Rotate the robot on launch -->
  <param name="SIM_TIMEOUT_SECONDS" type="int" value="$(optenv SIM_TIMEOUT_SECONDS 300)" />
  <param name="NAVIGATION_SUCCESS_COUNT" type="int" value="$(optenv NAVIGATION_SUCCESS_COUNT 1)" />
  <node pkg="test_nodes" type="navigation_test" name="navigation_test" output="screen"/>
</launch>
